scipy/optimize/_linprog_util.py

```diff
@@ -1372,18 +1319,18 @@ def _postsolve(x, postsolve_args, complete=False, tol=1e-8, copy=False):
     # if "complete", problem was solved in presolve; don't do anything here
     if not complete and bounds is not None:  # bounds are never none, probably
         n_unbounded = 0
-        for i, b in enumerate(bounds):
+        for i in range(bounds.shape[0]):
             if i in no_adjust:
                 continue
-            lb, ub = b
-            if lb is None and ub is None:
+            lbi = bounds[i,0]
```

Comment:
```suggestion
            lbi = bounds[i, 0]
```
Here and everwhere. I suggest checking your code with [flake8](https://flake8.pycqa.org/en/latest/) or a PEP8 style checker built into your IDE.